mnd/mnd/templates/rdrf_cdes/patient_event_handlers.html

```diff
@@ -69,11 +69,63 @@
             }
         });
 
+        $("#id_patient_insurance-dva_card_number").change(function() {
+            if (this.value.trim() == "") {
+                hide($("#id_patient_insurance-dva_card_type"), extraHideFn);
+            } else {
+                show($("#id_patient_insurance-dva_card_type"), extraRequiredFn);
+            }
+        });
+
+        $("#id_patient_insurance-referred_for_mac_care").change(function() {
```

Comment:
`referred_for_mac_care` compared to `3` again.

Reply:
See the previous comment.

Reply:
Updated.